cob_hardware_config/cob4-2/config/image_flip_params_sensorring_cam3d.yaml

```diff
@@ -0,0 +1,31 @@
+# rotation mode (0=fixed angle as defined  by rotation_angle, 1=automatic upright rotation against gravity using reference coordinate frame with upwards directed z-axis, 2=as 1 but image rotation only in 90deg steps -> faster)
+# int
+rotation_mode: 0
+
+# image rotation [in deg] (especially efficient with 0, 90, 180, 270 deg)
+# double
+rotation_angle: 90
+
+# reference world coordinate frame with z-axis pointing upwards (= against gravity direction)
+# string
+reference_frame: base_link
+
+# camera coordinate frame (image coordinate system with x-axis to the right, y-axis downwards, z-axis into viewing direction of the camera)
+# string
+camera_frame: cam3d_rgb_optical_frame
```

Comment:
this parameter looks wrong, but I get an error if I set it as sensorring_cam3d_frame...